### DISCUSSION TITLE
ConnectAPI: needs 2 functions for usage. Delete beta group and get builds for group.

### DISCUSSION THREAD

**Discussion by krish722** [fastlane/fastlane#17589] [General] [answered]
AppStoreConnectAPI way need 2 functions urgently

1. delete beta group. (testflight group)
2. get builds associated with beta group. (testflight group)

i have created a PR for the same https://github.com/fastlane/fastlane/pull/17588

Please review and help us guys!

**Comment by rogerluan** [comment, marked as answer]
https://github.com/fastlane/fastlane/pull/17588 has been merged 🎉 thanks for your contribution!
